pkgs/tools/misc/feed2mail/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, lib, python2Packages, fetchFromGitHub,
+configurationString ? null
+}:
+
+
+assert lib.asserts.assertMsg ( configurationString != null )
+  "You need to pass a configuration string to the feed2mail  package";
+
+let
+  configFile = builtins.toFile "feed2mail-config" configurationString;
+in
+python2Packages.buildPythonApplication rec {
+  pname = "feed2mail";
+  version = "2020-03-10";
+
+  src = fetchFromGitHub {
+    owner = "jonashaag";
+    repo = pname;
+    rev = "facd4d207bd7332f06b46b59a190710affa6c493";
+    sha256 = "0md75l93s03zxza8gmlqa9a9z88474wkk33h84wd7p9f91gvi0k3";
+  };
+
+  propagatedBuildInputs = with python2Packages; [ html2text feedparser ];
```

Comment:
Can we use python3?

Reply:
The package is still Python 2 only. I use it with Docker now so that I don't have to port the code to Python 3. I don't plan to port the code anytime soon. Happy to review and merge any PRs though!